CONTRIBUTING.md

```diff
@@ -1,34 +1,63 @@
-# Contributing
+# Wpt.fyi - Guidelines for Contributors
+Thank you for your interest in the Web Platform Tests Dashboard! You can participate by reporting issues and/or submitting bugs. (If you’d like to help maintain the project by submitting Pull Requests for new features, patches, or tests, please see our Maintainers’ Guidelines as well.)
```

Comment:
The scope of these documents have changed a bit since we started. Now, both bug reports *and* patches are considered "contributions" which anyone party may submit without special permission. "Maintaining" is focused on the folks who have some special rights/authority (i.e. the ability to merge changes into the repository).